src/SFA.DAS.Forecasting.Web/Controllers/EstimationController.cs

```diff
@@ -80,8 +80,8 @@ public ActionResult AddApprenticeships(string hashedAccountId, string estimation
 
             
         [HttpPost]
-        [Route("CalculateTotalCost", Name = "CalculateTotalCost")]
-        public async Task<ActionResult> CalculateTotalCost(string courseId, int numberOfApprentices, decimal? levyValue)
+        [Route("{estimationName}/apprenticeship/CalculateTotalCost", Name = "CalculateTotalCost")]
+        public async Task<ActionResult> CalculateTotalCost(string courseId, int numberOfApprentices, decimal? levyValue, string estimationName)
         {
             //TODO: this should be in the orchestrator
```

Comment:
Now that this method is using the orchestrator, can you remove this comment please.